README.md

```diff
@@ -35,6 +35,8 @@ await fastify.register(require('@fastify/swagger-ui'), {
   },
   staticCSP: true,
   transformStaticCSP: (header) => header,
+  transformSwagger: (swaggerObject, request, reply) => { return swaggerObject },
```

Comment:
Hmm, I think it better call `transformDocument` or `transformSpecification` as it is not only `swagger`.

Reply:
done